src/content/app/genome-browser/hooks/useGenomeBrowserTracks.ts

```diff
@@ -66,7 +71,9 @@ const useGenomeBrowserTracks = () => {
   useEffect(() => {
     // skip the effect if track categories have not yet been fetched
     // or if they have already been set
-    if (!trackCategories || !focusObject || trackSettingsForGenome) {
+    if (
+      !(trackCategories && !focusObject && trackSettingsForGenome && genomeId)
```

Comment:
`focusObject` here is getting negated twice here. I wonder if this makes the check a bit more confusing to read.

Reply:
I've remove the extra `!`